doc/examples/applications/plot_rank_filters.py

```diff
@@ -475,13 +487,18 @@
 # Here is an example of the classical morphological gray-level filters:
 # opening, closing and morphological gradient.
 
-from skimage.filters.rank import maximum, minimum, gradient
 
-noisy_image = img_as_ubyte(data.camera())
+noisy_image = ski.util.img_as_ubyte(ski.data.camera())
 
-opening = maximum(minimum(noisy_image, disk(5)), disk(5))
-closing = minimum(maximum(noisy_image, disk(5)), disk(5))
-grad = gradient(noisy_image, disk(5))
+opening = ski.filters.rank.maximum(
+    ski.filters.rank.minimum(noisy_image, ski.morphology.disk(5)),
+    ski.morphology.disk(5),
+)
+closing = ski.filters.rank.minimum(
+    ski.filters.rank.maximum(noisy_image, ski.morphology.disk(5)),
+    ski.morphology.disk(5),
+)
+grad = ski.filters.rank.gradient(noisy_image, ski.morphology.disk(5))
```

Comment:
Storing `ski.morphology.disk(5)` in an intermediate variable makes this a bit less verbose:
```suggestion
disk_5 = ski.morphology.disk(5)
opening = ski.filters.rank.maximum(
    ski.filters.rank.minimum(noisy_image, disk_5), disk_5,
)
closing = ski.filters.rank.minimum(
    ski.filters.rank.maximum(noisy_image, disk_5), disk_5,
)
grad = ski.filters.rank.gradient(noisy_image, disk_5)
```

Reply:
I took the liberty to push to your branch. Please feel totally welcome to revert or point out anything you don't agree with or understand! If you want to add more changes yourself, you should pull my commits to your local branch first. Don't hesitate to ask if you need help doing that. :blush:

Reply:
Hi, yes it makes sense. Thanks for making this change, I just checked the imports so I missed these :)